rules/windows/registry/registry_set/registry_set_office_trusted_location_uncommon.yml

```
@@ -0,0 +1,47 @@
+title: Uncommon Microsoft Office Trusted Location Added
+id: f742bde7-9528-42e5-bd82-84f51a8387d2
+related:
+    - id: a0bed973-45fa-4625-adb5-6ecdf9be70ac
+      type: derived
+status: experimental
+description: Detects changes to the registry keys related to "Trusted Location" of Microsoft Office where the path is set to something uncommon. Attacker might add additional trusted locations to avoid macro security restrictions.
+references:
+    - Internal Research
+    - https://admx.help/?Category=Office2016&Policy=excel16.Office.Microsoft.Policies.Windows::L_TrustedLoc01
+author: Nasreddine Bencherchali (Nextron Systems)
+date: 2023/06/21
+tags:
+    - attack.defense_evasion
+    - attack.t1112
+logsource:
+    category: registry_set
+    product: windows
+detection:
+    selection:
+        EventType: Setvalue
+        TargetObject|contains: 'Security\Trusted Locations\Location'
+        TargetObject|endswith: '\Path'
+    filter_exclude_known_paths:
+        Details|startswith:
+            - '%APPDATA%\Microsoft\Templates'
+            - '%%APPDATA%%\Microsoft\Templates'
+            - '%APPDATA%\Microsoft\Word\Startup'
+            - '%%APPDATA%%\Microsoft\Word\Startup'
+            - 'C:\Program Files (x86)\Microsoft Office\root\Templates\'
+            - 'C:\Program Files\Microsoft Office (x86)\Templates'
+            - 'C:\Program Files\Microsoft Office\root\Templates\'
+            - 'C:\Program Files\Microsoft Office\Templates\'
+    filter_main_office_click_to_run:
+        Image|startswith: 'C:\Program Files\Common Files\Microsoft Shared\ClickToRun\'
+        Image|endswith: '\OfficeClickToRun.exe'
+    filter_main_office_apps:
+        Image|startswith:
+            - 'C:\Program Files\Microsoft Office\'
+            - 'C:\Program Files (x86)\Microsoft Office\'
+        Image|endswith:
+            - '\winword.exe'
+            - '\excel.exe'
+    condition: (selection and (not 1 of filter_main_* and not 1 of filter_exclude_*))
```

```suggestion
    condition: selection and not 1 of filter_main_* and not 1 of filter_exclude_*
```
the grouping is not needed and imo doesn't help readability.